package.json

```diff
@@ -222,6 +224,7 @@
     "safe-buffer": "^5.2.0",
     "sdp-transform": "^2.12.0",
     "sinon": "^7.4.1",
+    "standard-changelog": "^2.0.24",
```

Comment:
Is there a reason why we are not using `devDependencies`?

Reply:
at this point it's a lost cause on the SDK

Reply:
Let's not add more code to the bundle though 🙁
or does the SDK not respect that too?

Reply:
It only bundles dependencies it touches, I'm pretty sure sinon isn't gonna show up in the final bundle since that only used in `test/`

Reply:
Also the SDK doesn't have a `devDependencies` property in the `package.json`
But you're more than welcome to sift through all the packages and figure out what's a devDependency and what's an actual dependency 😅